### ISSUE TITLE
unexpected item type 'AutotestRunner'

### ISSUE THREAD

**Issue by lantx97** [mapeditor/tiled#3822]
**Describe the bug**
I ran into this bug when I opened the project using qt 5.12.2. unexpected item type 'AutotestRunner'.

**Comment by Aayushmaan-24** [comment]
I have built Tiled locally with Qbs and am interested in tackling #3822 ("unexpected item type 'AutotestRunner'").

I am up for any guidance or suggestions on how to tackle this!

**Comment by bjorn** [comment]
@Aayushmaan-24 Hmm, probably this was just a case of using a too old version of Qbs? If you can confirm that this message doesn't appear anymore we can just close this issue. :-)

Minimum supported Qt version is also 5.15.2 in the meantime, 5.12 is no longer supported.

**Comment by Aayushmaan-24** [comment]
You're right — with Qt 5.15.2 (current minimum supported), the project parses fine in Qbs and Qt Creator. No "unexpected item type 'AutotestRunner'" error appears anymore.

Tested by:
- Opening tiled.qbs in Qt Creator (Qt 5.15 kit)
- Running `qbs build` (no parse errors)
- Autotests are configured and runnable.

Since Qt 5.12 is no longer supported, closing the issue makes sense. Happy to confirm further if needed!

Thanks!
